code/modules/jobs/job_types/captain.dm

```diff
@@ -105,6 +109,15 @@
 		return
 	celestial_charter.name_type = special_charter
 
+/datum/outfit/job/captain/plasmaman
+	name = "Captain (Plasmaman)"
+
+	uniform = /obj/item/clothing/under/plasmaman/captain
+	gloves = /obj/item/clothing/gloves/color/captain
+	head = /obj/item/clothing/head/helmet/space/plasmaman/captain
+	mask = /obj/item/clothing/mask/breath
+	r_hand= /obj/item/tank/internals/plasmaman/belt/full
```

Comment:
I think a lot more of these outfits may have formatting issues with `r_hand` so I'm not going to add reviews for each.